### PR TITLE
ethereum: Add setContractConfig for on-chain state

### PR DESCRIPTION
**Pull request by drozdziak1**
This method lets us alter the emitter/wormhole/chain IDs on-chain using the owner

### CODE REVIEW

**Review by ali-bahjati — APPROVED** [review]
lgtm!

File: ethereum/contracts/pyth/Pyth.sol [hunk 1]
```diff
@@ -73,14 +75,14 @@ contract Pyth is PythGetters, PythSetters, AbstractPyth {
         return info;
     }
 
-    function verifyPythVM(IWormhole.VM memory vm) private view returns (bool valid) {
+    function verifyPythVM(IWormhole.VM memory vm) private view returns (bool valid, string memory reason) {
         if (vm.emitterChainId != pyth2WormholeChainId()) {
-            return false;
+            return (false, "Invalid Chain ID");
```
Comment (drozdziak1) [inline ethereum/contracts/pyth/Pyth.sol:80]: This was a bit annoying to deal with when writing the test, and the existing error message was outdated anyway

File: ethereum/test/pyth.js [hunk 2]
```diff
@@ -362,6 +362,60 @@ contract("Pyth", function () {
...
+        let wrongChainIdFails = false;
+
+        try {
+            // Should result in expected failure
+            await updatePriceFeeds(this.pythProxy, [rawBatch]);
+        } catch (e) {
+            wrongChainIdFails = true;
+        }
+
+        assert.equal(wrongChainIdFails, true);
```
Comment (ali-bahjati) [inline ethereum/test/pyth.js:383]: I'm not sure what is the assert library here, but don't we have `assert.throws` or sth similar to make it more clear?

**Review by ali-bahjati — APPROVED** [review]

**Comment by drozdziak1** [comment]
This change is similar to #228 except for the wormhole on-chain field. We can revisit this if we think there's going to be a change to Wormhole's address

**Pull request closed without merging**